pkg/gen/nodejs-templates/yaml.ts.mustache

```diff
@@ -8,6 +8,7 @@ import fetch from "node-fetch";
 import * as k8s from "../index";
 import * as outputApi from "../types/output";
 
+
```

Comment:
Extra space intentional?  And is indentation intentional generally?

Reply:
I don't see any indentation changes.